.github/workflows/e2e-component-tests-embedding-sdk-cross-version.yml

```diff
@@ -228,12 +228,14 @@ jobs:
         with:
           script: | #js
             const {
+               generateReport,
                parseReport,
                formatSummary
              } = require("./.github/scripts/handle-mochawesome-report.js");
 
-             const report = parseReport();
-             const summary = formatSummary(report);
+             const report = generateReport();
+             const results = parseReport(report);
+             const summary = formatSummary(results);
```

Comment:
@iethree if we want to cheat a bit to make this diff smaller, we can generate and parse report in the same function. It's then up to you whether to (1) rename the function ~~`parseReport()`~~ -> `generateReport()` or to (2) leave the name as is (`parseReport()`).

It's all the same to me. Just LMK.

Reply:
P.S. This is all under the assumption that we don't really care about the file `./cypress/reports/cypress-test-report.json` being physically written to the file system. We were only using it to generate the summary, and that's what we now get with `merge({files}).then(report => do something with it)`.

`report` is a JS object so there's no need to first stringify it in order to create a JSON, and to then parse it back in order to consume it with `extractTestResultsInfo`.

I hope this makes sense?